esx_service/vmdk_ops.py

```diff
@@ -211,6 +220,14 @@ def validate_vsan_policy_name(policy_name, vmdk_path):
     if not vsan_policy.policy_exists(policy_name):
         raise ValidationError('Policy {0} does not exist'.format(policy_name))
 
+def validate_disk_allocation_format(format):
+    """
+    Ensure format is valid.
+    """
+    valid_opts = ["zeroedthick", "thin", "eagerzeroedthick"]
+    if not format.lower() in valid_opts:
+        raise ValidationError('Disk Allocation Format {0} does not exist'.format(format))
```

Comment:
I would update this to include valid options for e.g. 
Disk Allocation Format "xyz" does not exist. Valid options are: "thin" (default), "eagerzeroedthick", "zeroedthick"